index.bs

```diff
@@ -17,7 +17,10 @@ Abstract: This specification defines several new DOM events that provide informa
 Boilerplate: omit issues-index, omit conformance, repository-issue-tracking no
 Include MDN Panels: if possible
 Issue Tracking: DeviceOrientation Event Specification Issues Repository https://github.com/w3c/deviceorientation/issues
+Status Text: <p>This specification was initially published as a <a href="https://www.w3.org/TR/2016/CR-orientation-event-20160818/">Candidate Recommendation</a> in August 2016 and was <a href="https://www.w3.org/TR/2017/NOTE-orientation-event-20170530/">retired</a> in 2017 due to the Geolocation Working Group closure. In 2019 Devices and Sensors Working Group adopted this specification and during 2019-2024 made substantial interoperability, test automation, privacy and editorial improvements as outlined in the <a href="#changes">Changes</a> section. These changes aligned the specification with widely available implementations. In 2024 this specification became a joint deliverable between the Devices and Sensors Working Group and WebApps Working Group.</p><p>This document is maintained and updated at any time. Further changes are expected to be reflected in revised Candidate Recommendation Drafts and Snaphots as deemed appropriate to align the specification with existing implementations. Before requesting transition to Proposed Recommendation, the Working Groups will seek to demonstrate there is at least two independent, interoperable implementations of every feature defined in the specification, where interoperability can be verified by passing open test suites, and two or more implementations interoperating with each other.</p>
```

Comment:
Can we move the first paragraph to  a "Change log" or "History" section instead? 

The second section just states stuff that's already part of the W3C Process, no? I don't think it's necessary to restate what's part of the W3C Process or the Charter. 

For example, the spec is already going to link to: 
https://www.w3.org/standards/types/#x4-2-2-candidate-recommendation-snapshot

Reply:
The first section is a summary for busy people and SOTD is an appropriate place for this text. It may feel superfluous to you because you have tracked this specification closely. You are not the target audience :-) The [Changes](https://www.w3.org/TR/orientation-event/#changes) section complements this high-level summary and provides more details for readers such as yourself Marcos. Please review that section instead and let us know if you suggest any changes there.

It is important to document the success criteria here for this joint deliverable to avoid conflicts between the success criteria sections of the charters. Please let us know if you have suggestions for concrete changes to this text, otherwise we'll keep it as is. In particular we want to hear if there are any "at risk" features to document here.

Hearing no further concerns, I plan to merge this PR to prepare for the upcoming publication. Thank you for your support in making that happen.

Reply:
Wait! hold up. You're literally dismissing my feedback based on who I am? 

I'm literally an implementer of the specification and representing people who implement the specification. If the specification is not written for me as the intended audience, there is something very wrong here.

Reply:
I'm ambivalent on whether the full history in the first paragraph is worth including in the SOTD section or could be put in a separate "History" section. The link to the "Changes" section at a minimum assists new readers.

As discussed at TPAC the joint deliverables need to operate under the requirements of both working groups. It looks like the boilerplate above mentions both groups' patent policies but the lone "This document was produced by..." line only mentions one group. Similar to the patent policy text, I think a section linking to the "Success criteria" sections of both working group charters would be more precise and helpful than trying to summarize the intersection here.